src/noit_clustering.c

```diff
@@ -76,11 +76,14 @@ MTEV_HOOK_IMPL(noit_should_run_check,
 
 #define MAX_CLUSTER_NODES 128 /* 128 this is insanely high */
 #define REPL_FAIL_WAIT_US 500000
+#define DEFAULT_BATCH_SIZE 500
```

Comment:
```suggestion
#define DEFAULT_BATCH_SIZE 10000
```
This default is too small at this point, it only needs to be small in cases like where people have 50MB snmp configs.